### ISSUE TITLE
Logs and table preview retained after deleting all tables

### ISSUE THREAD

**Issue by nairabhishek73** [DalgoT4D/webapp#619]
https://jam.dev/c/1ee6fa7a-6aa2-496c-bd1c-31e59d00cd9a
https://jam.dev/c/b85deb82-71bd-4708-84ce-456b10147c2f

**Comment by fatchat** [comment]
The logs and preview disappear once the user clicks on the canvas

Dropping priority

**Comment by nairabhishek73** [comment]
Logs do not disappear when you click on the canvas. Will update priority on this later today if relevant after putting in all issues.

**Comment by fatchat** [comment]
You're right. 

i just realized: if they click on the canvas and the logs disappear, how do they get them back? for the preview they click on the node, for logs they would just be gone

**Comment by nairabhishek73** [comment]
Yeah at the moment i can only see the logs from the last run. Want to think this through and then i'll make the issue
